doc/protocols/scion-header.rst

```diff
@@ -408,3 +408,459 @@ Because of its special structure, no PathMeta header is needed. There is only a
+    |                        PacketTimestamp                        |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                          PathMetaHdr                          |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                           InfoField                           |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                              ...                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                           InfoField                           |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                           HopField                            |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                              ...                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                           HopField                            |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                              DVF                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+SCION Header Modifications
+--------------------------
+EPIC-SAPV contains the standard SCION header with the following 
+adaptations:
+
+   - Two reserved bits of the Meta Header are used to indicate the 
+     EPIC Version (EV).
+   - The size of the MAC (six bytes in standard SCION) inside the 
+     Hop Fields is reduced to two bytes, the four bytes of freed 
+     space are used for the Hop Validation Field (HVF). 
+
+Path Meta Header
+^^^^^^^^^^^^^^^^
+::
+
+     0                   1                   2                   3
+     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    | C |  CurrHF   |R| RSV |EV |  Seg0Len  |  Seg1Len  |  Seg2Len  |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ 
+
+EPIC Version (EV):
+   - **EV = 0:** *unused*
+   - **EV = 1:** *unused (in the future this may be used for EPIC-HP 
+     with improved path authorization on every AS on the path, which 
+     corresponds to EPIC Level 1 in the paper.)*
+   - **EV = 2:** Provides per-packet source authentication: every AS 
+     on the path can verify that the packet source is authentic. 
+     Corresponds to EPIC Level 2 in the paper.
+   - **EV = 3:** *unused (may be used for path validation in the 
+     future, which corresponds to EPIC Level 3 in the Paper.)*
+
+EPIC Reflected Response (R):
+  Unused at the moment. Meant to indicate that the packet is 
+  a (reflected) response to a previously sent EPIC Version 3 packet.
+
+Hop Field
+^^^^^^^^^
+We reduce the size of the MAC field to 2 bytes and assign a 4-byte 
+Hop Validation Field (HVF) to the freed space. The total size of the 
```

Comment:
Do these work the same way as in EPIC-HP? If yes, mention it. Otherwise, define how they work.

Reply:
They are not calculated the same way as in EPIC-HP.
The computation is specified later in the "Data plane (EPIC Version 2)" section.